tic-tac-toe-app/components/Game.jsx

```diff
@@ -0,0 +1,145 @@
+function Board() {
```

Comment:
There are three components here in Game.jsx. In order to organize your code in a more modular by separating the <Game/>, <Board/>, and <Square/> code into separate modules/files. Having these separated, logical chunks will make your code base more accessible and easier for other developers to reason about.